madgraph/iolibs/template_files/super_auto_dsig_group_v4.inc

```diff
@@ -114,23 +114,23 @@ C     Cannot make a selection with all PDFs to zero, so we return now
       ENDIF
       END
       
-      SUBROUTINE select_grouping(imirror,  iproc, iconf, wgt, nb_page)
+      SUBROUTINE select_grouping(imirror, iproc, iconf, wgt, VECSIZE_USED)
       USE DISCRETESAMPLER
       IMPLICIT NONE
 c     
 c     INPUT (VIA COMMAND BLOCK)
 c       SELPROC 
 c       SUMPROB
 c     INPUT
-c       nb_page (number of weight to update)
+c       VECSIZE_USED (number of weight to update)
 c     INPUT/OUTPUT
-c       WGTS(nb_page) #multiplied by the associated jacobian      
+c       WGTS(VECSIZE_USED) #multiplied by the associated jacobian      
 c
```

Comment:
typo here (not introduced by you)  it should be wgt and not wgts

Reply:
Ok I will change that

Reply:
Changed here https://github.com/mg5amcnlo/mg5amcnlo/pull/23/commits/dd98f006ce759c1d4d93aaeeaa20ecf080f6f9e0
Can be resolved